sdk/core/azure_core_amqp/Test-Setup.ps1

```diff
@@ -37,26 +37,26 @@ try {
   Invoke-LoggedCommand $cloneCommand
 
   Set-Location -Path "./azure-amqp/test/TestAmqpBroker"
-  Invoke-LoggedCommand "dotnet restore"
-  if (!$? -ne 0) {
-    Write-Error "Failed to restore dependencies for TestAmqpBroker."
-    exit 1
-  }
+  #  Invoke-LoggedCommand "dotnet restore"
+  #  if (!$? -ne 0) {
+  #    Write-Error "Failed to restore dependencies for TestAmqpBroker."
+  #    exit 1
+  #  }
+  #
+  #  Invoke-LoggedCommand "dotnet build"
 
-  Invoke-LoggedCommand "dotnet build"
+  Invoke-LoggedCommand "dotnet publish --self-contained --framework net6.0"
   if (!$? -ne 0) {
     Write-Error "Failed to build TestAmqpBroker."
     exit 1
   }
 
-  Invoke-LoggedCommand "dotnet publish --self-contained --framework net6.0"
-
   Write-Host "Test broker built successfully."
 
   # now that the Test broker has been built, launch the broker on a local address.
   $env:TEST_BROKER_ADDRESS = 'amqp://127.0.0.1:25672'
 
-  Write-Host "Starting test broker listening on " $env:TEST_BROKER_ADDRESS "..."
+  Write-Host "Starting test broker listening on $(env:TEST_BROKER_ADDRESS) ..."
```

Comment:
This won't work. It's not a subexpression. It's a variable:
```suggestion
  Write-Host "Starting test broker listening on ${env:TEST_BROKER_ADDRESS} ..."
```

Reply:
Again, unless you can figure out which version of PowerShell "fixed" it, putting `$env:TEST_BROKER_ADDRESS` *shouldn't* work. This is a compat issue. Putting a variable with delimiting characters in `${}` has always been the fix e.g., `"${env:TEST_BROKER_ADDRESS}"` or even `"${env:ProgramFiles(x86)}"`. When writing PowerShell scripts that could execute anywhere, it's always best to be explicit and take the lowest common denominator.

Reply:
IIRC, $(env:ProgramFiles) generated a parse error. I can try ${env:TEST_BROKER_ADDRESS}.